assignment-2/src/presenter/UsersListPresenter.js

```diff
@@ -0,0 +1,29 @@
+import model from "../model/UserModel";
+
+class UsersListPresenter {
+    onCreate() {
+        model.addUser(model.state.newUser.userName, model.state.newUser.password);
+        model.changeNewUserProperty("userName","");
+        model.changeNewUserProperty("password","");
+    }
+
+    onChange(property, value) {
+        model.changeNewUserProperty(property, value);
+    }
+    onLogin() {
+        //eval('debugger');
+        for(let index = 0; index < model.state.users.length; index++) {
```

Comment:
This for + if should be inside the model. The `window.location.assign` should stay here though.